### PR TITLE
Secure user with anonymized id

### PR DESCRIPTION
**Pull request by idegtiarov**
Currently not all template are in use, but they could be needed for next adaptive courses.

Urls are exchanged with relative protocol definition.

### CODE REVIEW

File: static/js/htmlframe.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
+<html>
+<head>
+
+	<script src="//ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js" type="text/javascript"></script>
+	<script src="//52.50.241.19/course-content/2014_T1/static/js/server_interaction.js" type="text/javascript"></script>
```
Comment (z4y4ts) [inline static/js/htmlframe.html]: Don't we have a domain name yet?

Comment (idegtiarov) [inline static/js/htmlframe.html]: Done

File: static/js/d3_histogram_plot.html [hunk 2]
```diff
@@ -355,38 +355,38 @@
 }
 
 function getUrlVars() {
-    var vars = {};
```
Comment (z4y4ts) [inline static/js/d3_histogram_plot.html]: Why change spaces to tabs?

File: static/js/histogram_entry_table.html [hunk 3]
```diff
@@ -18,12 +18,12 @@
   shape-rendering: crispEdges;
 }
 #section {
-    float:right;
```
Comment (z4y4ts) [inline static/js/histogram_entry_table.html]: Why?

File: static/js/d3_histogram_plot.html (hunk 2 above)
Comment (idegtiarov) [inline static/js/d3_histogram_plot.html]: tabs were set as initial indentation in parent file, but I agree difference in indentation it too annoying, and I prefer to use spaces in all new files in our project

**Comment by z4y4ts** [comment]
Wow, I didn't realize tabs to spaces conversion is going to be this massive. I'm afraid this is too error-prone.
Can you please instead revert last commit and add those files into `.editorconfig` as `indent_style = tab`so editors won't mix tabs and spaces anymore.

**Pull request merged**